packages/contracts/test/cyberDropBase.spec.ts

```diff
@@ -141,58 +133,63 @@ describe('CyberDropBase', function () {
           uri,
           timeStart,
           timeEnd,
-          price,
-          price,
-          stepDuration,
+          BigNumber.from(price),
           amountCap,
           shareCyber,
           signatureDrop
         )
 
       const tokenId = 0
+
       expect(
         await memory.contract.balanceOf(memory.other.address, tokenId)
-      ).to.eq('1')
+      ).to.eq(0x00)
       expect(
         await memory.contract.balanceOf(memory.oncyber.address, tokenId)
-      ).to.eq('0')
-      expect(await memory.contract.minterNonce(memory.other.address)).to.eq('1')
-      expect(await memory.contract.uri(tokenId)).to.eq(tokenURI(uri))
-      expect(await memory.contract['totalSupply()']()).to.eq('1')
-
-      const drop = await memory.contract.getDrop(tokenId)
-      expect(drop.timeStart).to.eq(timeStart)
-      expect(drop.timeEnd).to.eq(timeEnd)
-      expect(drop.priceStart).to.eq(price)
-      expect(drop.priceEnd).to.eq(price)
-      expect(drop.stepDuration).to.eq(stepDuration)
-      expect(drop.amountCap).to.eq(amountCap)
-      expect(drop.shareCyber).to.eq(shareCyber)
-      expect(drop.creator).to.eq(memory.other.address)
-      expect(drop.minted).to.eq('1')
+      ).to.eq(0x00)
 
-      await expect(
-        memory.contract.connect(memory.other).getDrop(1)
-      ).to.be.revertedWith('DNE')
+      expect(await memory.contract.minterNonce(memory.other.address)).to.eq(
```

Comment:
if drop doesn't exist function should throw an error not returning empty

Reply:
we should add 
```sol
require(drop.priceStart != 0, 'DNE');
```
in getDrop function to throw error if drop does not exists

Reply:
No we can't because some drops have a price of 0

Reply:
also because priceStart is not a thing anymore

Reply:
We can use amountCap though

Reply:
fixed